e2e/specs/clinical-forms.spec.ts

```diff
@@ -193,6 +193,7 @@ test('Form state is retained when minimizing a form in the workspace', async ({
   });
 
   await test.step('And I fill in values for the `White Blood Cells (WBC)`, `Platelets`, and `Neutrophils` questions', async () => {
+    await page.locator('#ManualInputWhiteBloodCells').waitFor();
```

Comment:
@denniskigen waiting for the Whit Blood Count locator to load before filling it out seems to prevent data loss.

Reply:
I've run the [GA](https://github.com/openmrs/openmrs-esm-patient-chart/actions/runs/10352931560) 5 times, and the test passed each time.

Reply:
Yep. Looks like that did the trick! I've run it a couple more times too. Thanks, @kdaud. We should probably document this somewhere.

Reply:
IIRC, the same behaviour occurs non-deterministically in the results viewer spec.

Reply:
> We should probably document this somewhere.

Not sure where to document this.

Reply:
That's fine, I'll sort it out. Mind approving the PR?